modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1176,54 +1179,70 @@ public List<Map<String, Serializable>> getValuesList(
 				extensionDynamicObjectDefinitionTable.getPrimaryKeyColumn()),
 			_EXPRESSIONS);
 
-		List<Object[]> rows = _list(
```

Comment:
It would be great to have a native way to modify dslqueries instead of doing this. I am thinking about something like a `DSLQuery visitor(DSLQuery query)` that transforms a query into another query modified. This way the contributors would just be visitors.

Reply:
Done